lib/globus/client/endpoint.rb

```diff
@@ -98,29 +83,58 @@ def paths
         end
       end
 
+      # @see #paths
+      def full_path
+        paths.last
+      end
```

Comment:
`paths.last` was showing up in three places, so I figured it was time to give it a name.

Reply:
New name is clearer, too.